src/App.scss

```diff
@@ -21,3 +21,32 @@
 .message-body {
   white-space: pre-line;
 }
+
+@media print, screen and (min-width: 769px) {
```

Comment:
The media query combines `print` and `screen and (min-width: 769px)`. Ensure this is intentional, as it affects both print and screen media types.